native/unittest/test_sort.c

```diff
@@ -0,0 +1,58 @@
+#include <assert.h>
+#include <string.h>
+#include "../sort.c"
+
+void test_heap_sort(MapPair* input, MapPair* expect, int n) {
```

Comment:
we can add more testcase, such as n =0, n=1, or n =large number. Otherwise, the key may be an empty string, may be chinese...